src/content/blog/astro-4120.mdx

```diff
@@ -61,6 +62,8 @@ Each island is loaded independent from the rest; this means a slower island, suc
 
 ### Performance
 
+<Note>[partialprerendering.com](https://www.partialprerendering.com/) has had some changes since this post was published. Rerunning the benchmarks maybe produce different results. We stand by these results as accurate the time of publishing.</Note>
```

Comment:
```suggestion
<Note>[partialprerendering.com](https://www.partialprerendering.com/) has had some changes since this post was published. Rerunning the benchmarks now may produce different results. These results were accurate at the time of publishing.</Note>
```

Reply:
oops, sorry i pushed before i saw this.

I think either is fine, no preference from me.

Reply:
Whichever you prefer is fine, but fix the maybe!